### PR TITLE
Script for fork_clone.py

### LINKED ISSUE #95: Fork and clone a repo
**Issue Description:**
A script to fork and clone a repo

**Expected Behavior:**
The script will have the code to fork a repository and then clone to it.

**Labels:**
Enhancement

### PR DESCRIPTION
**Pull request by GayathriManeksha**
## Fixes Issue

Closes #95 

## Changes proposed

Script to fork and clone a repo

## Screenshots
image

## Note to reviewers
git owner repo token name

owner: owner of the GitHub repo to be forked and cloned
repo: repository name
token: GitHub token
name: name for forked repo

### CODE REVIEW

File: scripts/fork_clone.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+parser.add_argument('owner', type=str, help='Repository owner')
+parser.add_argument('repo', type=str, help='Repository name')
+parser.add_argument('token', type=str, help='GitHub API token')
+parser.add_argument('name',type=str,help='Enter name for forked repo')
+args = parser.parse_args()
+
+# Get the command-line arguments
+owner = args.owner
+repo = args.repo
+token = args.token
```
Comment (sahil-sagwekar2652) [inline scripts/fork_clone.py:19]: we already have a github_secrets.py file in the project. import the token from there.

Comment (GayathriManeksha) [reply scripts/fork_clone.py]: I used the github_secrets.py to import the token
image

**Comment by sahil-sagwekar2652** [comment]
Please check the linter output, and make the necessary chanes. I'll merge this PR asap.

**Review by sahil-sagwekar2652 — APPROVED** [review]
Great work. Thanks for contributing. I am merginG this PR.

**Pull request merged**